app/routes/task_routes.py

```diff
@@ -0,0 +1,93 @@
+from flask import Blueprint, jsonify, abort, make_response, request
+from app import db
+from app.models.task import Task
+from app.routes.routes_helpers import validate_model, create_msg_slack
+import datetime
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+
+
+
+#POST endpoint CREATES A NEW TASK
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+        request_body = request.get_json()
+        try:
+            new_task = Task.from_dict(request_body)
+        except KeyError:
+            return {
+                "details": "Invalid data"
+            }, 400
+        db.session.add(new_task)
+        db.session.commit()
+
+        return {f"task": new_task.to_dict()}, 201
+
+#GET ALL TASKS endpoint
+@tasks_bp.route("", methods=["GET"])
+def read_all_tasks():
+    sort = request.args.get("sort")
+
+    if sort == "asc":
+        tasks = Task.query.order_by(Task.title.asc())
+    elif sort == "desc":
+        tasks = Task.query.order_by(Task.title.desc())
+    else:
+        tasks = Task.query.all()
+
+    task_force = [task.to_dict() for task in tasks]
+    return jsonify(task_force), 200
```

Comment:
Let's make sure we keep consistent organization and space between routes
```suggestion
    return jsonify(task_force), 200
    
```